### DISCUSSION TITLE
CRUD - Datime - Chrome - Firefox

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#1184] [Q&A (Help)] [answered]
I've this field on seupOperation:

```
        $this->crud->addField([
            'name' => 'start',
            'label' => trans('backpack::ash_base.start_date'),
            'type' => 'datetime',
            'value' => $currentUserModel->start ?? null,
            'tab' => trans('backpack::ash_base.working_hours'),
            'wrapper' => [
                'class' => 'form-group col-md-6'
            ],
        ]);
```

On Chrome i see:
<img width="909" alt="Screenshot 2024-10-09 alle 15 26 08" src="https://github.com/user-attachments/assets/536dacc4-80a4-4fad-9e64-692236eb1cdb">

On Firefox i see:
<img width="880" alt="Screenshot 2024-10-09 alle 15 25 14" src="https://github.com/user-attachments/assets/f93401d1-ad50-4731-bd76-f971c544b40d">

here time selector & "today" link are missing !!

How can i fix it ?

**Comment by karandatwani92** [comment, marked as answer]
Hey @blondie63 

The `datetime` [field](https://backpackforlaravel.com/docs/6.x/crud-fields#datetime-1) is browser-dependent.

If you need a browser-independent `datetime` field customized in your way, Try `datetime_picker` [field](https://backpackforlaravel.com/docs/6.x/crud-fields#datetime_picker-pro).

**Comment by blondie63** [comment]
@karandatwani92 please take a look here
<img width="786" alt="Screenshot 2024-10-24 alle 13 35 20" src="https://github.com/user-attachments/assets/77ba4ccd-f2cc-4c71-82e9-15ebf08fa480">
